test/dynamo/test_export.py

```diff
@@ -3023,7 +3023,7 @@ def f_non_list_operands(x):
         example_inputs = (torch.rand(5),)
         with self.assertRaisesRegex(
             RuntimeError,
-            "Expect operands to be a tuple of Tensors, but got",
+            r"Expect operands to be a tuple of pytrees that only consists of tensor leaves, but got",
```

Comment:
nit: The word "pytree" may confuse users. A better error message is something like "Expected operands to only contain Tensors"